### DISCUSSION TITLE
IVF_SQ8 showing memory leakage

### DISCUSSION THREAD

**Discussion by marait123** [milvus-io/milvus#23233] [Q&A and General discussion]
## environemnt
kubernetes 

- Milvus version:
2.2
- Deployment mode(standalone or cluster):
cluster
- MQ type(rocksmq, pulsar or kafka):    
kafka
- SDK version(e.g. pymilvus v2.0.0rc2):
pymilvus 
- OS(Ubuntu or CentOS): 
ubuntu

## description

hello guys I was trying to measure memory usage by query node after loading a collection  with 2 million vectors with ivf_sq8 indexing. as milvus started loading the collection memory increased from 355MB to 1.333GB 

after releasing the collection from memory. memory decreased only to 1.159GB

![image](https://user-images.githubusercontent.com/44650320/230211605-117348e0-b45b-48d7-bc6e-9fffd3d28bf3.png)

## what I expect

this behavior shows some kind of memory leakage in the app. I expect memory to at least go back to the same amount as it was before loading the collection when the collection is released.

**Comment by yhmo** [comment]
Let me do some local testing.

**Reply by yhmo** [reply]
You are right. I can reproduce it, after load() and release(), the memory is not freed. 
I have created an issue: https://github.com/milvus-io/milvus/issues/23278
